README.md

```diff
@@ -69,10 +60,21 @@ The GraphQL Gateway, configured in `gateway.tf`, serves as the central entry poi
 4. **Initialize Terraform**: Run `terraform init` to initialize the Terraform workspace.
 5. **Apply Configuration**: Execute `terraform apply` to deploy the resources to your Kubernetes cluster.
 
+#### Auto-updating Build
```

Comment:
you should mention that you still need to kill the pods if you want them to update